controllers/api.js

```diff
@@ -164,15 +164,38 @@ router.post('/auth', postLimiter, async function (req, res) {
   if (req.body.refresh_token) {
     // need to refresh token
     if (await u.loadByRefreshToken(req.body.refresh_token)) {
-      res.send({ refresh_token: u.getRefreshToken(), access_token: u.getAccessToken() });
+      const body = {
```

Comment:
Successful OAuth2 token response body:
```json
{
  "access_token":"...",
  "token_type":"bearer",
  "expires_in":3600,
  "refresh_token":"...", // don't include in body if null
}
```
- Don't need to send refresh token expiry.
- `config.auth...` props should have sensible defaults if `undefined`?